lib/v2/ezone/index.js

```diff
@@ -45,7 +46,7 @@ module.exports = async (ctx) => {
 
```

Comment:
Fix lazy load img
https://github.com/DIYgod/RSSHub/pull/7274/files#diff-1e51bc387121e57607de3fe8502fe533ba696d6be5048e016d0a645f0319fa61R49-R55

Reply:
Would you also remove `.ulvideo`?